styles/base.css

```diff
@@ -5,5 +5,7 @@ body {
     font-size: 100%;
     color: #313131;
     text-align: center;
+    width: 480px;
+    margin: 0 auto;
```

Comment:
check your CSS indentation, indentation should be 2 spaces, just like in JavaScript (it's incorrect in all these files)